test/common.js

```diff
@@ -0,0 +1,147 @@
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import crypto from 'crypto';
+import fs from 'fs';
+import path from 'path';
+import { spawn } from 'child_process';
+import sinon from 'sinon';
+
+const
+	DEFAULT_HASH_ALGORITHM = 'sha256',
+	DEFAULT_HASH_ENCODING = 'hex',
+	TEST_CACHE_PATH = 'test/sandbox/test-cache',
+	TEST_CLIENT_ID = '982f0d7f23a94811849f703ef8150401',
+	TEST_TEMP_PATH = 'test/sandbox/test-temp',
+	TEST_SHARED_SECRET = '68JLP0L6M10/kfvtXNQCrbXe3nz8g1smyCjTblcy4OKbd4cnxf0anuxvLZ0T4hZw8hMi8w8pPvezd6AcBwa8Hw==';
```

Comment:
## This is a grave mistake.

Functioning PlayNetwork API credentials have been publicly exposed on the Internet as a result of this pull request.

This issue must be considered with extreme seriousness - for the time from when this commit was pushed to GitHub until the secret for the clientId referenced in this PR was changed, there was a huge amount of risk present to PlayNetwork made open and public to the Internet. While we have corrected this on the backend and we will chalk this up to a learning experience this time around, I've worked in companies where people would have been let go for this type of breach because it borders on gross negligence. 

Any code submitted to a public repository requires an extra layer of in-depth checks in the code *WELL IN ADVANCE* of performing even a local commit occurs - this fact was spelled out clearly in the story item and discussed verbally prior to the commencement of this work. Even if one were to have committed this locally and then removed it on a subsequent commit, the history of these commits and the changes are published to GitHub and, in this case, these credentials still would have been exposed.